docs/upgrading.md

```diff
@@ -0,0 +1,73 @@
+# Upgrading the Agent
+
+This document provides user guidance around agent upgrades and versioning.
```

Comment:
Not sure what this means. Which agent? Is it user guidance for upgrading the agent? Is it user guidance for applying version information to the agent, or for something else?

Reply:
You could, for example, write

"When you install and use an instance of the Splunk Distribution of OpenTelemetry Java, you need to set a plan for upgrading your instance and assigning useful version numbers to it."

Reply:
> You could, for example, write
> 
> "When you install and use an instance of the Splunk Distribution of OpenTelemetry Java, you need to set a plan for upgrading your instance and assigning useful version numbers to it."

I like that this is prescriptive, but "instance" isn't completely appropriate here and the topic here isn't really about the user assigning version numbers.